.github/workflows/checks.yml

```diff
@@ -20,16 +24,23 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
+        with:
+          # TODO delete
+          fetch-depth: 0
 
       - name: Install Node
         uses: actions/setup-node@v3
         with:
           node-version-file: .nvmrc
-          cache: "yarn"
+          cache: yarn
 
       - name: Install dependencies
         run: yarn install --frozen-lockfile --prefer-offline
 
+      - name: Check for uncommitted changes
+        run: |
+          git diff --name-only --exit-code --relative || (echo "\nUncommitted changes found up there 👆" && exit 1)
```

Comment:
it verifies that there are no uncommited yarn.lock changes

Reply:
nice